src/compiler/crystal/syntax/ast.cr

```diff
@@ -643,14 +643,18 @@ module Crystal
       end
     end
 
-    def self.new(obj, name, arg : ASTNode)
-      new obj, name, [arg] of ASTNode
+    def self.new(obj, name, arg : ASTNode, global = false)
+      new obj, name, [arg] of ASTNode, global: global
     end
 
     def self.new(obj, name, arg1 : ASTNode, arg2 : ASTNode)
       new obj, name, [arg1, arg2] of ASTNode
     end
 
+    def self.new(obj, name, arg1 : ASTNode, arg2 : ASTNode, arg3 : ASTNode)
```

Comment:
I know at this point we _could_ maybe use a splat, but I'd like to avoid creating intermediate tuples. 0..3 is enough.